src/main.cpp

```diff
@@ -1362,9 +1372,26 @@ int main(int argc, char *argv[])
 
 	if (!S9xGraphicsInit ())
          erk();
+	// read command line options
+	for (i = 1; i < argc; i++) {
+		if (!strcmp(argv[i], "-rom")) {
```

Comment:
remove this option, load ROM by default and allow only for 1 argument or output stderr